packages/client/components/TeamPrompt/TeamPromptDrawer.tsx

```diff
@@ -73,13 +86,18 @@ const TeamPromptDrawer = ({meetingRef, isDesktop}: Props) => {
     })
   }
 
-  // Render the discussion thread if it can be rendered, otherwise fall back on the work sidebar.
-  // :TRICKY: Elements rendered with JSX never return null, so we need to render both possible
-  // internal drawers via function calls in order to nullish coalesce. We also have to call both
-  // every time for the React hooks to be consistent.
-  const renderedDiscussionDrawer = TeamPromptDiscussionDrawer({meetingRef: meeting, onToggleDrawer})
-  const renderedWorkDrawer = TeamPromptWorkDrawerRoot({meetingRef: meeting, onToggleDrawer})
-  const renderedInnerDrawer = renderedDiscussionDrawer ?? renderedWorkDrawer
```

Comment:
i'd rather fetch more data than use a hacky way of rendering components. this isn't guaranteed to work in a future versions of react and may have unintended consequences